Examples/Scripts/Python/itk_seeding.py

```diff
@@ -0,0 +1,171 @@
+        forwardSeedConfirmationRange=acts.SeedConfirmationRange(
+            3000.0, -3000.0, 140.0, 1, 2
+        ),
+    )
+
+    seedFilterConfig = acts.SeedFilterConfig(
+        maxSeedsPerSpM=seedFinderConfig.maxSeedsPerSpM,
+        deltaRMin=seedFinderConfig.deltaRMin,
+        impactWeightFactor=100,
+        compatSeedWeight=100,
+        compatSeedLimit=3,
+    )
+
+    seedingAlg = acts.examples.SeedingAlgorithm(
+        level=acts.logging.VERBOSE,
+        inputSpacePoints=[evReader.config.outputSpacePoints],
+        outputSeeds="PixelSeeds",
+        outputProtoTracks="prototracks",
+        gridConfig=gridConfig,
+        seedFinderConfig=seedFinderConfig,
+        seedFilterConfig=seedFilterConfig,
+        zBinNeighborsTop=[
+            [0, 0],
+            [-1, 0],
+            [-1, 0],
+            [-1, 0],
+            [-1, 0],
+            [-1, 1],
+            [0, 1],
+            [0, 1],
+            [0, 1],
+            [0, 1],
+            [0, 0],
+        ],  # allows to specify the number of neighbors desired for each bin, [-1,1] means one neighbor on the left and one on the right, if the vector is empty the algorithm returns the 8 surrounding bins
+        zBinNeighborsBottom=[
+            [0, 1],
+            [0, 1],
+            [0, 1],
+            [0, 1],
+            [0, 1],
+            [0, 0],
+            [-1, 0],
+            [-1, 0],
+            [-1, 0],
+            [-1, 0],
+            [-1, 0],
+        ],
+    )
+
+    s = s or acts.examples.Sequencer(
+        events=1, numThreads=-1, logLevel=acts.logging.INFO
+    )
+
+    s.addReader(evReader)
+    s.addAlgorithm(seedingAlg)
+
+    return s
+
+
+if "__main__" == __name__:
+
+    # create temporary file
+    with tempfile.TemporaryDirectory() as tmpdirname:
+        temp = open(tmpdirname + "/event000000000-spacepoints_pixel.csv", "w+b")
+        print(
+            "created temporary file: "
+            + tmpdirname
+            + "/event000000000-spacepoints_pixel.csv"
+        )
+        temp.write(
+            b"measurement_id,sp_type,module_idhash,sp_x,sp_y,sp_z,sp_radius,sp_covr,sp_covz\n 1,0,3139,32.67557144165039,-5.311902523040771,-47.65000152587891,33.10452270507812,0.05999999865889549,0.02999880164861679\n 2,0,3422,95.14442443847656,-15.46361255645752,-52.125,96.39286804199219,0.05999999865889549,0.01687432639300823\n 3,0,3650,102.8257064819336,-16.71612739562988,-52.67499923706055,104.1755981445312,0.05999999865889549,0.001875000074505806\n 4,0,4223,159.4266204833984,-25.91166687011719,-56.75,161.5186157226562,0.05999999865889549,0.02999880164861679\n 5,0,5015,224.07958984375,-36.37123107910156,-61.40000152587891,227.0121765136719,0.05999999865889549,0.007499700412154198\n 6,0,6023,284.1485595703125,-46.0638542175293,-65.72499847412109,287.8580932617188,0.05999999865889549,0.001875000074505806"
+        )
```

Comment:
Neat! If you don't want to use `b""` here, I think you can use

```python
temp = open(tmpdirname + "/event000000000-spacepoints_pixel.csv", "w+t")
```

in line 156 to open the file in text mode directly.

Reply:
done